code/accounts/program-derived-address/create-a-pda/program/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
Could we get any folder called `target` removed at the main `.gitignore`, instead of a new `.gitifnore` for this directory?

Reply:
done